showcase/src/app/shared/table-of-contents/_table-of-contents-theme.scss

```diff
@@ -0,0 +1,7 @@
+.docs-toc-container {
```

Comment:
This class is coming from Angular implementation. Here, we developed a "Table of contents", maybe should we have a related class name ? 😊
What do you think about this ?